packages/components/card/template.njk

```diff
@@ -31,7 +35,7 @@
       {%- endif %}
     </h{{ headingLevel }}>
   {%- if params.type %}
-    <span class="nhsuk-card--care__arrow" aria-hidden="true"></span>
+    <span class="nhsuk-card--care__primary" aria-hidden="true"></span>
```

Comment:
Is this "nhsuk-card--care__primary" class used anywhere?

Reply:
This was a mistake, and it shouldn't have been changed from `nhsuk-card--care__arrow`, I've reverted it back now.